packages/stories-app/src/login.jsx

```diff
@@ -230,22 +230,44 @@ export function LoginDialog(props) {
             setShowLogin(0);
         }
     }
-
+    const handleKeypressLogin = e => {
+        // listens for enter key
+      if (e.keyCode === 13) {
```

Comment:
@ronran1 keyCode is deprecated. I know this was accepted, but want to fix this whenever you make your next commit?